IPython/lib/display.py

```diff
@@ -323,7 +323,7 @@ def __init__(self,
         ----------
         path : str
             path to the file or directory that should be formatted
-        directory_prefix : str
+        url_prefix : str
             prefix to be prepended to all files to form a working link [default:
             'files']
```

Comment:
@takluyver Just playing around with FileLink now, and spotted that too.

Will add to this PR.